openshift/templates/agent/agent-deploy.bc.prod.param

```diff
@@ -12,14 +12,14 @@
 # SUFFIX=-bc
 # APP_NAME=orgbook
 # APP_GROUP=aries-vcr
-APPLICATION_DOMAIN=orgbook-agent.apps.silver.devops.gov.bc.ca
-APPLICATION_ADMIN_DOMAIN=orgbook-agent-admin.apps.silver.devops.gov.bc.ca
+APPLICATION_DOMAIN=agent.orgbook.gov.bc.ca
+APPLICATION_ADMIN_DOMAIN=agent-admin.orgbook.gov.bc.ca
```

Comment:
Same name-syncing comment as for other environments.

Reply:
updated